tooling/lint-package-names.js

```diff
@@ -0,0 +1,53 @@
+// const shelljs = require(`shelljs`);
+const fs = require(`fs`);
+
+const modules = [];
+
+
+// JS equivelent of "set -e" bash command
+// for (var envVariable in process.env ) {
```

Comment:
In the original script the set -e command was the first command run but it never printed outputted anything when run locally. In the JS equivelent here, it lists out ALL environment variable. In the interest of matching the original output from the .sh script i commented this part out but did not wanted to remove it completely if this was in fact intended behavior. Is the outputted list of all environment variables on the machine intended behavior here?